apps/ledger-live-mobile/src/components/Stake/index.tsx

```diff
@@ -59,9 +59,6 @@ const StakeFlow = ({ route }: Props) => {
     },
     [navigation, parentRoute],
   );
-  const onError = (error: Error) => {
-    logger.critical(error);
-  };
```

Comment:
I think we need restore this.

Reply:
IMO you don't and that's the whole point of the PR, if you are able to find any other message than the one we want to remove I'm happy to do this PR another way but currently there is only on error message sent here and it's `Request account interrupted by user` nothing else.

Did you check my comment above ? (https://github.com/LedgerHQ/ledger-live/pull/3767#issuecomment-1600839757)

I've checked all usage of `NavigatorName.RequestAccount` and we never set `params.error`, so there is no chance of any other error landing in your onError fonction, I've also check all other navigator setting `beforeRemove` and none is using the `params.error` so again no possibility of any other message.

I've also seen that your implementation in `Stake/index.tsx` is the only one setting the `onError` callback outside the `params` of the screen, it's okay and worked because the object itself represents the params of the navigator but it was not matching the other places in our codebase.

Reply:
@Justkant , what is it the problem to listen here them?

Reply:
I've explained the problem multiple times already

> currently there is only on error message sent here and it's `Request account interrupted by user` nothing else

I've not implemented this logic, it's here from a long time ago, no idea who started that pattern.
No one but the platform and wallet-api is actually using this (like I said already too) and I've said that we want to change that in the future as it's not a good solution to send errors back

Reply:
@sarneijim Can you please list all message that are triggered here other than the one listed in sentry notifications ?

If there is nothing else, it's dead code. Dead code should be removed

Reply:
If onError(route.params.error) has not been removed, onError callback is not "dead code".